
float32 depth_raw_unit